verilog/formatting/formatter_test.cc

```diff
@@ -1605,7 +1605,54 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      "      .in (ex_input_pins.in)\n"  // aligned
      "  );\n"
      "endmodule\n"},
-
+    {"module test;\n"
+     "bind entropy_src tlul_assert #(.EndpointType(\"Device\"))\n"
+     "tlul_assert_device (.clk_i, .rst_ni, .h2d(tl_i), .d2h(tl_o));\n"
+     "endmodule\n",
+     "module test;\n"
+     "  bind entropy_src tlul_assert #(\n"
+     "      .EndpointType(\"Device\")\n"
+     "  ) tlul_assert_device (\n"
+     "      .clk_i,\n"
+     "      .rst_ni,\n"
+     "      .h2d(tl_i),\n"
+     "      .d2h(tl_o)\n"
+     "  );\n"
+     "endmodule\n"},
+    {"module test;\n"
+     "bind entropy_src tlul_assert #(.EndpointType(\"Device\"))\n"
+     "tlul_assert_device (.clk_i, .rst_ni,\n\n .h2d(tl_i),\n\n .d2h(tl_o));\n"
+     "endmodule\n",
+     "module test;\n"
+     "  bind entropy_src tlul_assert #(\n"
+     "      .EndpointType(\"Device\")\n"
+     "  ) tlul_assert_device (\n"
+     "      .clk_i,\n"
+     "      .rst_ni,\n"
+     "\n"
+     "      .h2d(tl_i),\n"
+     "\n"
+     "      .d2h(tl_o)\n"
+     "  );\n"
+     "endmodule\n"},
+    {"module test;\n"
+     "bind entropy_src tlul_assert #(.EndpointType(\"Device\"))\n"
+     "tlul_assert_device (\n//c1\n.clk_i, .rst_ni,\n //c2\n .h2d(tl_i),\n "
+     "//c3\n .d2h(tl_o));\n"
+     "endmodule\n",
+     "module test;\n"
+     "  bind entropy_src tlul_assert #(\n"
+     "      .EndpointType(\"Device\")\n"
+     "  ) tlul_assert_device (\n"
+     "      //c1\n"
+     "      .clk_i,\n"
+     "      .rst_ni,\n"
+     "      //c2\n"
+     "      .h2d(tl_i),\n"
+     "      //c3\n"
+     "      .d2h(tl_o)\n"
+     "  );\n"
```

Comment:
That was an unfortunate choice of code to test as we don't see what is happening to the alignment here.

Could you add tests with comments and empty lines with ports with names of different sizes? This way we will actually see if they are getting aligned or not

Reply:
BTW what happens if there is a port without `()`? Can we add such test?

And shouldn't we always align to those anyway ports anyway? (this is just a question, please don't change the implementation without any ACK that we should be doing this - or maybe there is something similar in the style guide?)